packages/@okta/vuepress-site/.vuepress/nav/concepts.js

```diff
@@ -0,0 +1,56 @@
+// the navigation that is shown across the reference pages
```

Comment:
Because this is a mid point in the conversion to the new prose layout across the board, this is a temporary solution on the sidebar.  We will be migrating to a more dynamic way of handling sidebar in the future, but for now, it is hard coded because we need to still support the old sidebar as well.

Reply:
Do you mean like include a 'categorization' metadata within the index.md file for each guide/concept/reference that would be programmatically extracted and then assign the content to an appropriate spot in relevant navigation tree(s)? Presumably we'd want to do that at a daily/nightly build time, not at run time, otherwise the user would be hit with time costs whenever they access the pages.

Reply:
It would mean there is no `navigation/*.js` file for generating the sidebar nav,  it would build the nav based on folder structure when `yarn build` happens

Reply:
Awesome. That future sounds great, I'm all for iterative improvement.